src/containers/App/App.js

```diff
@@ -1,36 +1,86 @@
 import './App.css';
-import { getDefaultData }from '../../ApiFetch'
-import React, { Component } from 'react'
-import { connect } from 'react-redux'
-import { getAllMovies } from '../../actions'
+import {
+  getDefaultData
+} from '../../ApiFetch';
+import React, {
+  Component
+} from 'react';
+import {
+  connect
+} from 'react-redux';
+import {
+  getAllMovies
+} from '../../actions';
+import Gallery from '../../components/Gallery/Gallery';
+import UserMenu from '../UserMenu/UserMenu';
+import {
+  Route,
+  Link
+} from 'react-router-dom';
 
 export class App extends Component {
-
   async componentDidMount() {
-    const movieData = await getDefaultData()
-    console.log('movieData', movieData)
-    getAllMovies(await movieData)
+    const results = await getDefaultData();
+    await this.props.getAllMovies(results);
   }
 
   render() {
-    return (
-      <div>
-        
-      </div>
-    )
+    return ( <
+      Route exact path = '/'
+      render = {
```

Comment:
This prettier thing confuses me a bit it looks like it broke everything onto its own line.